Sources/KSCrashRecording/include/KSCrashReportFields.h

```diff
@@ -24,219 +24,228 @@
+KSCRF_DEFINE_CONSTANT(KSCrashField, MemoryTermination, memoryTermination, "memory_termination")
+
+KSCRF_DEFINE_CONSTANT(KSCrashField, CrashedThread, crashedThread, "crashed_thread")
+KSCRF_DEFINE_CONSTANT(KSCrashField, AppStats, appStats, "application_stats")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BinaryImages, binaryImages, "binary_images")
+KSCRF_DEFINE_CONSTANT(KSCrashField, System, system, "system")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Memory, memory, "memory")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Threads, threads, "threads")
+KSCRF_DEFINE_CONSTANT(KSCrashField, User, user, "user")
+KSCRF_DEFINE_CONSTANT(KSCrashField, ConsoleLog, consoleLog, "console_log")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Incomplete, incomplete, "incomplete")
+KSCRF_DEFINE_CONSTANT(KSCrashField, RecrashReport, recrashReport, "recrash_report")
+
+KSCRF_DEFINE_CONSTANT(KSCrashField, AppStartTime, appStartTime, "app_start_time")
+KSCRF_DEFINE_CONSTANT(KSCrashField, AppUUID, appUUID, "app_uuid")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BootTime, bootTime, "boot_time")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BundleID, bundleID, "CFBundleIdentifier")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BundleName, bundleName, "CFBundleName")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BundleShortVersion, bundleShortVersion, "CFBundleShortVersionString")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BundleVersion, bundleVersion, "CFBundleVersion")
+KSCRF_DEFINE_CONSTANT(KSCrashField, CPUArch, cpuArch, "cpu_arch")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BinaryCPUType, binaryCPUType, "binary_cpu_type")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BinaryCPUSubType, binaryCPUSubType, "binary_cpu_subtype")
+KSCRF_DEFINE_CONSTANT(KSCrashField, DeviceAppHash, deviceAppHash, "device_app_hash")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Executable, executable, "CFBundleExecutable")
+KSCRF_DEFINE_CONSTANT(KSCrashField, ExecutablePath, executablePath, "CFBundleExecutablePath")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Jailbroken, jailbroken, "jailbroken")
+KSCRF_DEFINE_CONSTANT(KSCrashField, KernelVersion, kernelVersion, "kernel_version")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Machine, machine, "machine")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Model, model, "model")
+KSCRF_DEFINE_CONSTANT(KSCrashField, OSVersion, osVersion, "os_version")
+KSCRF_DEFINE_CONSTANT(KSCrashField, ParentProcessID, parentProcessID, "parent_process_id")
+KSCRF_DEFINE_CONSTANT(KSCrashField, ProcessID, processID, "process_id")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Size, size, "size")
+KSCRF_DEFINE_CONSTANT(KSCrashField, Storage, storage, "storage")
+KSCRF_DEFINE_CONSTANT(KSCrashField, SystemName, systemName, "system_name")
+KSCRF_DEFINE_CONSTANT(KSCrashField, SystemVersion, systemVersion, "system_version")
+KSCRF_DEFINE_CONSTANT(KSCrashField, TimeZone, timeZone, "time_zone")
+KSCRF_DEFINE_CONSTANT(KSCrashField, BuildType, buildType, "build_type")
+
+KSCRF_DEFINE_CONSTANT(KSCrashField, MemoryFootprint, memoryFootprint, "memory_footprint")
```

Comment:
Should we also split `KSCrashField` into smaller groups?

Reply:
I was thinking about it, but I didn't want to break the names.